pcweb/pages/index/demos/charts/charts.py

```diff
@@ -0,0 +1,285 @@
+                    "fontFamily": "var(--font-instrument-sans)",
+                    "fontSize": "0.9rem",
+                    "fontStyle": "normal",
+                    "fontWeight": "500",
+                    "lineHeight": "1rem",
+                    "letterSpacing": "-0.01094rem",
+                    "minWidth": "8rem",
+                    "padding": "6px 10px",
+                    "position": "relative",
+                    "::after": {
+                        "content": "''",
+                        "position": "absolute",
+                        "left": "0",
+                        "top": "0",
+                        "width": "4px",
+                        "height": "100%",
+                        "backgroundColor": "var(--c-violet-9)",
+                        "borderTopLeftRadius": "0.75rem",
+                        "borderBottomLeftRadius": "0.75rem",
+                    },
+                },
+                item_style={
+                    "color": "currentColor",
+                    "display": "flex",
+                    "paddingBottom": "0px",
+                    "justifyContent": "space-between",
+                },
+                label_style={"color": "var(--c-slate-12)"},
+                cursor={
+                    "strokeWidth": 1,
+                    "stroke": "var(--slate-a5)",
+                    "fill": "var(--slate-a5)",
+                },
+                separator="",
+            ),
+            rx.recharts.cartesian_grid(
+                horizontal=True, vertical=False, class_name="!stroke-slateA-3"
+            ),
+            rx.recharts.x_axis(
+                # interval="preserveStartEnd",  # This prop doesn't exist in the current reflex version
+                include_hidden=True,
+                data_key="month",
+                stroke="currentColor",
+                type_="category",
+                class_name="!font-small text-slate-9 [&>line]:!text-slateA-3",
+            ),
+            data=ChartsState.data,
+            class_name="w-full h-full overflow-visible",
+        ),
+        class_name="flex flex-col items-end gap-6 px-10 py-12 h-full overflow-hidden",
+    )
+
+
+charts_code = """data = [
+    {
+        "month": "Jan",
+        "Mobile": random.randint(100, 500),
+        "Desktop": random.randint(400, 700),
+    },
+    ...
+]
+
+import reflex as rx
+
+class ChartsState(rx.State):
+    data = data
+
+    def randomize_data(self):
+        self.data = [
+            {
+                "month": item["month"],
+                "Mobile": random.randint(100, 500),
+                "Desktop": random.randint(400, 700),
+            }
+            for item in self.data
+        ]
+
+def chart():
+    return rx.box(
+        rx.button(
+            "Randomize",
+            on_click=ChartsState.randomize_data,
+            class_name="button",
+        ),
+        rx.recharts.area_chart(
+            rx.el.svg.defs(
```

Comment:
I think this looks good visually, but could it scare people into thinking making graphs is very complicated in reflex? I wonder if we can have the code be for a simple styled graph while still showing the fancy one